src/common/queries/projects.ts

```diff
@@ -45,26 +45,3 @@ export function useProjectQuery(params: { id: string | undefined }) {
     { staleTime: Infinity }
   );
 }
-
-export function useFetchProjectQuery() {
```

Comment:
@beganovich The hook is not used in any part of the app's logic, so I removed it.